osu.Game.Rulesets.Mania/Timing/Drawables/DrawableScrollingTimingChange.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Game.Rulesets.Timing;
+
+namespace osu.Game.Rulesets.Mania.Timing.Drawables
+{
+    public class DrawableScrollingTimingChange : DrawableManiaTimingChange

```

Comment:
But... It's in osu.Game.Rulesets.Mania? I thought we weren't going to use the ruleset in the name unless they conflicted with something in osu.Game?